templates/modern/src/toc.ts

```diff
@@ -54,7 +54,7 @@ export async function renderToc(): Promise<TocNode[]> {
     if (node.href) {
       const url = new URL(node.href, tocUrl)
       node.href = url.href
-      active = normalizeUrlPath(url) === normalizeUrlPath(window.location)
+      active = isExternal(url) ? false : normalizeUrlPath(url) === normalizeUrlPath(window.location)
```

Comment:
Consider add an external URL node to `samples/seed/articles/toc.yml` as a test case. Snapshot tests should then make sure we don't regress this bug in the future.